src/app/Fake.Tools.Squirrel/Squirrel.fs

```diff
@@ -0,0 +1,143 @@
+/// Contains types and utility functions related to creating [Squirrel](https://github.com/Squirrel/Squirrel.Windows) installer.
+[<RequireQualifiedAccess>]
+module Fake.Tools.Squirrel
+
+open Fake.Core
+open Fake.IO
+open Fake.IO.FileSystemOperators
+open Fake.IO.Globbing
+open System
+open System.IO
+open System.Text
+
+/// The [Squirrel](https://github.com/Squirrel/Squirrel.Windows) Console Parameters type.
+///
+/// For reference, see: [Squirrel Command Line Options](https://github.com/Squirrel/Squirrel.Windows/blob/master/docs/advanced-releasify.md)
+type ReleasifyParams =
+    {
```

Comment:
maybe we can use the F# guideline formatting here? ie start the first comment on the `{` line?
But you can leave it as is, just a suggestion

Reply:
I'm on the "unusual in F#" side of the fence, a code style not aligned on 4 characters boundaries where I need to type whitespaces manually in my editor doesn't suit me :)

(Also I just didn't touch indentation from the original module)

Reply:
> (Also I just didn't touch indentation from the original module)

Yes but we try to improve things when we port, this is the only opportunity to improve and break stuff ;)

Reply:
I will use the also acceptable ` = {` style because it looks a lot better ;)

Reply:
At least I tried :(